src/send_proc.py

```diff
@@ -15,148 +15,147 @@
 chore_struct = utils.readStructFromJSON("chores.json")
 
 
-def run() -> None:
-    hasUpdated = notCompleted = False 
-    violationSent = True
-    notCompleted = True
-    isViolationTime = lambda time: time == 0 or time % 15 == 0 or time % 30 == 0 or time % 45 == 0
+def runSchedule() -> None:
+	global assigned
+	lazies: list = None
+	hasUpdated = notCompleted = violationSent = punishTime = False 
+	isViolationTime = lambda time: time == 0 or time % 15 == 0 or time % 30 == 0 or time % 45 == 0
```

Comment:
```suggestion
	isViolationTime = lambda time: time.minute % 15 == 0
```

Reply:
yikes so much not needed code

Reply:
Pretty sure `minute == 0` is required though since 0 % 15 would equal 15

Reply:
fc8506f

Reply:
Nope, 0%15 = 0. See for yourself in your python env

Reply:
```
Python 3.6.6 (default, Jul 21 2020, 23:21:26)
[GCC 4.2.1 Compatible Apple LLVM 10.0.1 (clang-1001.0.46.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> print(0 % 15)
0
```